pkg/cmd/cmd.go

```diff
@@ -0,0 +1 @@
+package cmd
```

Comment:
correct :(  
Basically gives a "no files to compile error", I believe there is one or two in origin also